open-craft/opencraft
# [BB-2446] Added a message on console if the emails not been verified

**Issue**: [BB-2446](https://tasks.opencraft.com/browse/BB-2446)

**Screenshots**(Updated):
![image](https://user-images.githubusercontent.com/8607745/82952908-72b51600-9fc7-11ea-8b17-394d5e238a47.png)

**Testing Instructions**:
- Create a new user, in starting it won't have the verified, the alert should be showing on the console.
- Next, verify the mail and the info should not show on the console.

**Reviewers**:
